src/core/lib/experiments/experiments.yaml

```diff
@@ -149,3 +149,11 @@
   owner: alishananda@google.com
   test_tags: []
   allow_in_fuzzing_config: false
+- name: chttp2_allow_config_override_keepalive_permit_without_calls
+  description:
+    Allows overriding keepalive_permit_without_calls.
+    Refer https://github.com/grpc/grpc/pull/33428 for more information.
+  expiry: 2023/07/07
```

Comment:
This expiry is very short... maybe give it a month or two?

Reply:
yeah ok, from what i understand, this is mostly for the bot

Reply:
it's to make sure these things get deleted eventually (plenty of codebases exist where that's not the case)

Reply:
done